precompiles/balances-erc20/src/tests.rs

```diff
@@ -319,7 +319,10 @@ fn transfer() {
 #[test]
 fn transfer_not_enough_funds() {
 	ExtBuilder::default()
-		.with_balances(vec![(CryptoAlith.into(), 1000)])
+		.with_balances(vec![
+			(CryptoAlith.into(), 1000),
+			(CryptoBaltathar.into(), 1000),
```

Comment:
AFAICT baltathar is not needed?

Reply:
Baltathar is needed to reflect the correct behavior of the test. If we only leave Alith, we encounter some discrepancy with the `TotalIssuance`, having some `AritmeticUnderflow` error instead of the error that we are looking for. With Baltathar there, the TotalIssuance is greater than Alith's balance so we don't have trouble to test the desire functionality. This change was introduced in v0.9.43 and it's related to Currency deprecation.